unity-client/Assets/Scripts/MainScripts/DCL/Components/LoadableShapes/NFTShape/NFTShape.cs

```diff
@@ -45,7 +44,7 @@ protected override void AttachShape(DecentralandEntity entity)
 
             entity.OnShapeUpdated += UpdateBackgroundColor;
 
-            LoadWrapper_NFT loadableShape = GetOrAddLoaderForEntity<LoadWrapper_NFT>(entity);
+            var loadableShape = GetOrAddLoaderForEntity<LoadWrapper_NFT>(entity);
```

Comment:
Just curiosity: Why have you changed the `LoadWrapper_NFT` type by the generic `var`?

Reply:
To be honest, I don't know... `loadableShape` was a also a member field and I had to make it local in other part of the code and while doing that I unnecessary changed that in the process... no real reason